samples/RefreshFlow/AngularApp/angular-cli.json

```diff
@@ -1,39 +1,41 @@
 {
   "project": {
-    "version": "1.0.0-beta.20-4",
-    "name": "openiddict-samples.refresh-flow.angularapp"
+    "version": "1.0.0-beta.26",
+    "name": "angularapp"
   },
   "apps": [
     {
       "root": "src",
-      "outDir": "./wwwroot",
+      "outDir": "dist",
       "assets": [
         "assets"
       ],
       "index": "index.html",
       "main": "main.ts",
+      "test": "test.ts",
```

Comment:
@Toxicable Yes. It does make sense to leave that in if it has a useful impact on the sample.